### ISSUE TITLE
curl support

### ISSUE THREAD

**Issue by toddaa** [stackery/php-lambda-layer#30]
Looking for some direction here.  I'm attempting to enable curl support and didn't know where better to ask this.

I'm trying to create a secondary layer which includes my bootstrap script.  This consists of the following:
```
#!/bin/sh
cd $LAMBDA_TASK_ROOT
/opt/bin/php /opt/runtime.php
```
The runtime.php file is there get the body of the message sent to the Lambda function which I am trying to do so with curl and then invoke the functions handler.  Unfortunatly this doesnt work.  I've tried to put a php.ini file in the function root (/var/task) containing `extension=curl.so`, but I still dont seem to have access to it.  Here's the files current state which simply tests for curl:
```
<?php
if  (in_array  ('curl', get_loaded_extensions())) {
    echo "CURL is available";
} else {
    echo "CURL is not available";
}
?>
```
I saw a few other options in the README about how to include the other modules, but I dont know if I am able to use them without spinning up docker and creating my own PHP layer.  Any advise?

**Comment by toddaa** [comment]
As another test, I've moved the runtime.php file out of the secondary layer and placed it into the function root.  It now lives along side the php.ini file.  I changed the bootstrap script to call it in `/var/task` rather than `/opt`.  Still no curl.

**Comment by txase** [comment]
Hi @toddaa!

You shouldn't create your own bootstrap when using this runtime layer. That effectively would make your own runtime, which may or may not work with how this layer is meant to operate :).

If you're trying to figure out how to use the curl extension in your own PHP scripts on top of this runtime, then all you should need to do is create a package with two files:

- php.ini
    ```ini
    extension=curl.so
    ```
- index.php
    ```php
    <?php
    if  (in_array  ('curl', get_loaded_extensions())) {
        echo "CURL is available";
    } else {
        echo "CURL is not available";
    }
    ?>
    ```

I hope this helps! I'm going to close the issue as I believe there's nothing here to fix. Please feel free to reopen if you find a bug in how the layer operates. 🙏

**Comment by toddaa** [comment]
@txase, 
Thanks for the comment.  Believe it or not, you gave me exactly the information I needed to connect these dots.  In my case, I guess I do need to create my own runtime.  I thought I could just use this for my needs to forgo maintaining my own.
Thanks again for taking the time!

**Comment by toddaa** [comment]
@txase,
I forgot to mention that in addition to curl I was also looking to add zip.  While testing I did dump both the 7.3 and 7.1 module directories.  I noticed that the current 7.3 layer you have doesnt seem to be including the zip.so file along with a few others listed in your README.  Here's the directoy list for 7.3 I have from my Cloudwatch logs:
```
bz2.so
calendar.so
ctype.so
curl.so
exif.so
fileinfo.so
ftp.so
gettext.so
iconv.so
json.so
mbstring.so
mysqli.so
mysqlnd.so
pdo.so
pdo_mysql.so
pdo_pgsql.so
pdo_sqlite.so
pgsql.so
phar.so
sockets.so
sqlite3.so
tokenizer.so
```
